baselines/powerbi.md

```diff
@@ -1,26 +1,27 @@
+**`TLP:CLEAR`**
+
 # CISA M365 Security Configuration Baseline for Power BI
 
-Power BI is the Software as a Service (SaaS) offering from Microsoft that facilitates self-service business intelligence dashboards, reports, datasets, and visualizations. Power BI can connect to multiple, different data sources, combine and shape data from those connections, then create reports and dashboards to share with others. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Power BI security.
+Power BI is the software as a service (SaaS) product from Microsoft that facilitates self-service business intelligence dashboards, reports, datasets, and visualizations. Power BI can connect to multiple different data sources, combine and shape data from those connections, then create reports and dashboards to share with others. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Power BI security.
```

Comment:
Change first sentence to:
Microsoft 365 (M365) Power BI is a cloud-based product that facilitates self-service business intelligence dashboards, reports, datasets, and visualizations. 

The "Microsoft 365 (M365)" is required to match the standard. The "is a cloud-based" may not apply to Power BI but is the standard phase used in the other SCBs

Reply:
Thanks for the re-word looks good